GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -351,35 +356,59 @@ private void tvGitTree_AfterSelect(object sender, TreeViewEventArgs e)
             Task ViewItem()
             {
                 return e.Node?.Tag is GitItem gitItem
-                    ? ViewGitItemAsync(gitItem)
-                    : Task.CompletedTask;
+                    ? ShowGitItemAsync(gitItem)
+                    : ClearOutputAsync();
             }
+        }
 
-            Task ViewGitItemAsync(GitItem gitItem)
+        private Task ShowGitItemAsync(GitItem gitItem)
+        {
+            switch (gitItem.ObjectType)
             {
-                switch (gitItem.ObjectType)
-                {
-                    case GitObjectType.Blob:
-                    case GitObjectType.Commit:
+                case GitObjectType.Blob:
                     {
-                        GitItemStatus file = new(name: gitItem.FileName)
+                        if (!blameToolStripMenuItem1.Checked)
```

Comment:
What do you think about not using this logical negation and instead writing `blameToolStripMenuItem1.Checked == false`? Would be more readable, you can miss that `!`

Reply:
Also, in the method `ExecuteCommand` in `RevisionDiffControl,cs`, you use (reuse) one style of switch statements (making cases inline) and here you do in one style, FYI.

Reply:
> What do you think about not using this logical negation and instead writing blameToolStripMenuItem1.Checked == false? Would be more readable, you can miss that !

In this case, I prefer the compact style

> Also, in the method ExecuteCommand in RevisionDiffControl,cs, you use (reuse) one style of switch statements (making cases inline) and here you do in one style, FYI.

Is this about how Blame is activated?
In that case it is intentional, FileTree remembers Blame/View for those that prefer to only view (that is faster).
For RevDiff it is always a one time activation, next selection is always Diff (or New etc) again.

Reply:
> What do you think about not using this logical negation and instead writing `blameToolStripMenuItem1.Checked == false`? Would be more readable, you can miss that `!`

This is C# intrinsic (and C-like languages in general). The only time one would write ` == false` when the left operand is a nullable.